Development/src/pages/nodes.js

```diff
@@ -228,7 +229,7 @@ const ItemArrayField = ({ className, source, record = {} }) => (
     <div>
         {get(record, source).map(item => (
             <div key={item} className={className}>
-                {item}
+                <Typography>{item}</Typography>
```

Comment:
Is the outer `div` still required or could the `key` and `className` be added to the `Typography`?